components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultServiceURLBuilder.java

```diff
@@ -466,28 +463,31 @@ protected void appendContextToUri(StringBuilder serverUrl, String contextPath) {
         }
     }
 
-    private boolean setUrlForOrgPerspectiveAccess(StringBuilder resolvedUrlStringBuilder) {
+    private void setUrlBasedOnOrgOrTenantPerspective(StringBuilder resolvedUrlStringBuilder, String tenantDomain) {
```

Comment:
change method name to setURL

Reply:
Fixed by - https://github.com/wso2/carbon-identity-framework/pull/5148/commits/9210f866727e96ccfb5abef34fc2eab3b90ac60e